app/proguard-rules.pro

```diff
@@ -94,9 +94,10 @@
 
 -dontwarn com.google.errorprone.annotations.*
 
-# work around a bug in proguard
-# see https://sourceforge.net/p/proguard/bugs/729/
--keepnames public interface com.uber.autodispose.lifecycle.CorrespondingEventsFunction { *; }
```

Comment:
This old rule can be removed as apparently R8 does not have this problem like ProGuard did